android/app/src/main/java/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -109,6 +116,8 @@ else if (layoutMode == LayoutMode.planning)
     if (mToggleMapLayerButton != null)
     {
       mToggleMapLayerButton.setOnClickListener(view -> mMapButtonClickListener.onMapButtonClick(MapButtons.toggleMapLayer));
+      if (TrafficManager.INSTANCE.isEnabled() || IsolinesManager.isEnabled() || SubwayManager.isEnabled() || Framework.nativeIsOutdoorsLayerEnabled())
+        mToggleMapLayerButton.setColorFilter(Color.BLUE);
```

Comment:
best to move this to a `updateLayerButton` function, then call it every time a layer is enabled/disabled in the popup menu

Reply:
Yep, can you please help?

Reply:
Maybe instead of changing color a number of active layers can be displayed? 1, 2 or 3? It's a temporary solution, but it should be visible to users to attract their attention.

Or maybe abbreviate layer names with words like O(outdoors), E(elevation), S(subway)? But then in different languages it will be difficult.